admin/class-create-block-theme-admin.php

```diff
@@ -469,6 +469,7 @@ function get_theme_slug( $new_theme_name ) {
 
 		$old_slug = wp_get_theme()->get( 'TextDomain' );
  		$new_slug = sanitize_title( $new_theme_name );
+		$new_slug = preg_replace('/\s+/', '', $new_slug); // Remove spaces
```

Comment:
Not sure if I am missing a usecase, isn't it taken care by `sanitize_title`?

Reply:
`sanitize_title` doesn't remove the white spaces, we need to remove them to create a slug.